### ISSUE TITLE
Do we need Google Group for discussion?

### ISSUE THREAD

**Issue by piyanatk** [prickly-pythons/prickly-pythons#2]
I was thinking about using GitHub issue tracking as a question/discussion platform, but there seem to be no easy way to open an issue with an email, which many people will probably find more useful.

**Comment by kpolsen** [comment]
Can we use this 'Issues' option on GitHub as a forum for any question? In stead of a google group? I get an email notification for each new issue that you post, which is nice at the moment, but might of course be too many emails at some point...

**Comment by piyanatk** [comment]
Yes, the problem is that you can only open a new issue by going to github page. No option to send an email to open an issue.

Regarding too many email, that is where filtering option in your email client become helpful

**Comment by kpolsen** [comment]
OK I see. Yeah, I will need to do some filtering later on. For now, are we only notified when a new issue comes up or if we commented on that issue? Or does everybody >1 notification for this issue even if they never commented?

**Comment by spacegal-spiff** [comment]
Just to keep everyone posted, I emailed Github support if there was Google Groups-like functionality for people to email the group directly, and here was their response (reverse chronological order):

---

Hey Kim,

Great thanks for clarifying - though we can't make any promises about feature requests, we'll share this with the team for you.

Thanks again!
Robert

Me:     Thanks for the quick reply! That's exactly what our group was thinking -- a
    quick way to communicate within the organization, without having to go to
    Github directly. We were thinking of something sort of similar to how
    Google Groups handles discussions and replies through email (but were
    hoping to have it all in one spot on Github). It looks like it's possible
    to reply to issues directly via email, so having similar functionality to
    start a discussion or an issue via email would be really great!

Hi Kim,

Thanks for getting in touch! There's no generic email address like you've described associated with organization accounts. Just to clarify, is ease of use the reason why you're interested in something like this? E.g. a student wouldn't have to go to github.com and open an issue to discuss something with the group, they can just write an email?

Thanks,
Robert

Me: 
    We have just set up a new Github Organization (https://github.com/sese-python-users) for our astronomy department, so students can post questions and code and work collaboratively toward improving their programming skills. It would be ideal if organization members could send an email directly from their personal email accounts to our organization at large. For example, when push/pull/issue notifications occur on Github, emails are sent to the organization members. Is there a "generic" email address corresponding to our organization? Something like "sese-python-users@github.com"?

**Comment by kpolsen** [comment]
Thanks Kim for following that through - sounds like they're considering this feature...
